docs/main/user-guides/migration/legacy-storage-to-new-storage/migration-overview.md

```diff
@@ -0,0 +1,27 @@
+---
+id: migration-overview
+title: Migration from Legacy Storage to New Storage
+keywords:
+ - Migration Overview
+description: This section outlines the process of migrating the legacy storage to new storage.
+---
+
+# Migration Overview
+
+In this migration process, we are using [pv-migrate](https://github.com/utkuozdemir/pv-migrate) that is a CLI tool/kubectl plugin to easily migrate the contents of one Kubernetes `PersistentVolumeClaim` to another.
```

Comment:
Let's give an introduction paragraph here, of why we need this section.
Try to keep it generic, like from any non-OpenEBS storage to OpenEBS storage.